pkg/didcomm/protocol/outofband/service.go

```diff
@@ -169,6 +171,46 @@ func (s *Service) HandleOutbound(_ service.DIDCommMsg, _, _ string) error {
 	return errors.New("not implemented")
 }
 
+// AcceptRequest from another agent and return the connection ID.
+func (s *Service) AcceptRequest(r *Request) (string, error) {
+	connID, err := s.handleRequestCallback(&callback{
+		msg:      service.NewDIDCommMsgMap(r),
+		myDID:    "",
+		theirDID: "",
+	})
+	if err != nil {
+		return "", fmt.Errorf("failed to accept request : %w", err)
+	}
+
+	return connID, err
+}
+
+// SaveRequest created by the outofband client.
+func (s *Service) SaveRequest(r *Request) error {
+	// TODO the ID when saving this request...
+	err := s.connections.SaveInvitation(r.ID+"-TODO", r)
```

Comment:
Please, add more description and create an issue for that. I don't understand the problem.

Reply:
done: #1547